src/services/camUser/data.ts

```diff
@@ -0,0 +1,53 @@
+import * as tencentcloud from 'tencentcloud-sdk-nodejs'
+import { ClientConfig } from 'tencentcloud-sdk-nodejs/tencentcloud/common/interface'
+import CloudGraph from '@cloudgraph/sdk'
+import groupBy from 'lodash/groupBy'
+import isEmpty from 'lodash/isEmpty'
+import { SubAccountInfo } from 'tencentcloud-sdk-nodejs/tencentcloud/services/cam/v20190116/cam_models'
+import loggerText from '../../properties/logger'
+import { TencentServiceInput } from '../../types'
+import { initTestEndpoint, generateTencentErrorLog } from '../../utils'
+
+const lt = { ...loggerText }
+const { logger } = CloudGraph
+export const serviceName = 'CamUser'
+const apiEndpoint = initTestEndpoint(serviceName)
+
+export interface RawTencentCamUser extends SubAccountInfo {
+  id: string
+  region: string
+}
+
+export default async ({
+  regions,
+  config,
+}: TencentServiceInput): Promise<{
+  [region: string]: RawTencentCamUser[]
+}> =>
+  new Promise(async resolve => {
+    const camUserList: RawTencentCamUser[] = []
+
+    for (const region of regions.split(',')) {
+      try {
+        const CamClient = tencentcloud.cam.v20190116.Client
+        const clientConfig: ClientConfig  = { credential: config, region, profile: { httpProfile: { endpoint: apiEndpoint } } }
+        const cam = new CamClient(clientConfig)
+
+        const response =  await cam.ListUsers(null)
+
+        if (response && !isEmpty(response.Data)) {
+          for (const instance of response.Data) {
+            camUserList.push({
+              id: `${region}-${instance.Uid}`,
```

Comment:
@tyler-dunkel , the api returns a same user list with different regions. So the id of the user is made of ${region}-${uid}

Reply:
Hmm. Can we filter the users out by region instead of changing their ids?

Reply:
hey @tyler-dunkel what do you mean about filter the users out by region?
What I see so far from the returned data is something like the below structure. The user list seems duplicates for all the regions. But I can't guarantee the list would always be the same. Didn't find any info from the doc.

Region-A:
 - user 1
 - user 2
 - user 3
 Region-B:
 - user 1
 - user 2
 - user 3
 Region-C:
 - user 1
 - user 2
 - user 3

Reply:
Does the client require `region` as a param when querying for the users or is there any way to determine which region a user belongs to? there is a chance that the users are global (i.e. not belonging to one region) like they are in AWS. In that case, I would set the region to `global` and filter the users into one list based on their id. We should avoid having duplicates or altering the id returned by the sdk where possible.

Reply:
@tyler-dunkel yeah, make sense. 
Just tested, the `region` is optional in the config object and the sdk does return a list of users without the region. 
Filter the users with the same id into the `global` list could be tricky. For example a user is in 2 out of 3 regions, we can't add it to the `global` list.
I read their document and find a step by step screenshot to create a new user. You can check [here](https://intl.cloud.tencent.com/zh/document/product/267/32468). It is in Chinese, but you can find the first picture in the page. It has `Beijing Center` on the top right. So is that means create user would create a new user just in that region or across all regions?
To conclude, I would say the users are all in `global` region, although Tencent seems doesn't have `global`  region.  In our case, I don't think the client created those 4 users in all 20+ regions, since the ids are the same. I will do the `global` approach for now,  meanwhile we should keep this in mind when more/real data come in.

Reply:
its a good question regarding if users end up being region specific (in some cases) or are always global to all regions. In regards to `although Tencent seems doesn't have global region.` Similar to some other providers (we do this in aws I think), just create a "fake" global region for the users to be put in. I.e. the users field `region` = `global` but there is no `global` option in the region list for when they run `cg init tencent`

Reply:
@tyler-dunkel I have the `global` update pushed. Also resolved the merge conflicts.